src/components/PersonLink.tsx

```diff
@@ -0,0 +1,81 @@
+import cn from 'classnames';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  personSlug?: string;
+  persons: Person[];
+};
+
+export const PersonLink: React.FC<Props> = ({
+  persons,
+  person,
+  personSlug,
+}) => {
+  const {
+    sex,
+    name,
+    motherName,
+    fatherName,
+    slug,
+    born,
+    died,
+  } = person;
+  const isFemale = person.sex === 'f' ? 'has-text-danger' : '';
+  const selectPerson
+ = slug === personSlug
+ || motherName === personSlug
+  || fatherName === personSlug ? 'has-background-warning' : '';
```

Comment:
Try to fix writing of this const. Something like this. And move father ternary into a variable.
```suggestion
  const selectPerson = slug === personSlug
    || motherName === personSlug
    || fatherName === personSlug ? 'has-background-warning' : '';
```